themes/digital.gov/src/scss/new/guides/_guide-menu-bar.scss

```diff
@@ -54,6 +43,23 @@
     display: none;
   }
 
+  &-links a {
```

Comment:
Override `<a>` tags for desktop or mobile styles.

Reply:
We should add a class to these links if possible.

I see the mobile nav is rendering empty class attribute:
![image](https://github.com/GSA/digitalgov.gov/assets/3385219/5210cb8b-9b9d-4267-8223-830b3dc8b955)